test/fixtures/BootstrapCommand/version-mismatch/packages/package-2/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "package-2",
+  "version": "1.0.0",
+  "dependencies": {
+    "package-1": "^1.0.0",
```

Comment:
The mismatches happen after a few prereleases in a row, with some packages getting changes and some not. Eventually the versions stop matching and without warning get packages from the registry.

We're sometimes doing prereleases for minor or even patches. Could that be a reason too?

Irregardless, I'd like a flag to always link, never pull internal packages from registry. Sounds like @mbecke wants such a flag as well..